content/en/docs/installation/kubernetes.md

```diff
@@ -114,10 +114,10 @@ In order to install the Helm chart, you must follow these steps.
 Install the `CustomResourceDefinition` resources separately.
 ```bash
 # Kubernetes 1.15+
-$ kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.14.0/deploy/manifests/00-crds.yaml
+$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/untagged-681c789ba37426bee486/cert-manager.crds.yaml
```

Comment:
These need to point to the public facing URLs 🙈 even if they don't exist yet publicly

Reply:
oh no